mesh/mesh.go

```diff
@@ -131,14 +131,13 @@ func rlpHash(x interface{}) (h common.Hash) {
 	return h
 }
 
-func SerializableTransaction2StateTransaction(tx *types.SerializableTransaction) *Transaction {
-	price := big.Int{}
-	price.SetBytes(tx.Price)
+func SerializableSignedTransaction2StateTransaction(tx *types.AddressableSignedTransaction) *Transaction {
+	price := &big.Int{}
```

Comment:
can use big.NewInt(value) instead of these two lines

Reply:
I set it to a uint64 not int64 as the constructor's param